blog/2022-1-2-your-first-extension.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: your-first-extension
+title: Your first extension
+authors: [CharlieS1103]
+tags: [spicetify, documentation, community,development]
+---
+
+## So you want to make an extension....
+
+
+### First step:
+Start out by setting up your environment: 
+In terminal or powershell, run ``spicetify enable-devtool apply``
+Ensure that you have a text editor or IDE at the ready.
+
+
+### Second step:
+First, run spicetify config-dir, this should open up file explorer or finder(dependent on your OS), next, open the "Extensions folder" and create a file titled extension.js, or whichever title you choose(For this tutorial we will be using extension.js). Open this file, and input the following code:
```

Comment:
Missing the \` in run `spicetify config-dir`.

Edit: Firstly, run `spicetify config-dir`, this should open up your default file manager. Secondly, open the `Extensions` folder and create a file titled `extension.js`, or whichever title you choose, for this tutorial we will be using extension.js for consistency purposes. Finally, open this file, and paste the following code: